azure-pipelines/compliance.yml

```diff
@@ -13,100 +13,114 @@ pr:
   autoCancel: true
   branches:
     include:
-      - main
-      - dev/*
+    - main

```

Comment:
YAML is whitespace sensitive, so isn't this indentation change causing the block to be wrong?

Reply:
I think the old code had incorrect spacing. Looking at the schema the '-' is same column as include (like the change in this pr). See the code sample at the bottom:
https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines

Reply:
Ah, interesting. Great then.